packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.component.tsx

```diff
@@ -58,6 +60,18 @@ const VitalsHeader: React.FC<VitalsHeaderProps> = ({ patientUuid }) => {
     const vitalsTakenToday = Boolean(dayjs(latestVitals?.date).isToday());
     const vitalsOverdue = hasActiveVisit && !vitalsTakenToday;
 
+    const now = dayjs();
+    const vitalsOverduePeriod = Math.round(dayjs.duration(now.diff(latestVitals?.date)).asDays());
+    let overdueVitalsTag: string = '';
+
+    if (vitalsOverduePeriod >= 1 && vitalsOverduePeriod <= 7) {
+      overdueVitalsTag = `These vitals are ${vitalsOverduePeriod} days old.`;
```

Comment:
Should we find something for the 1-day old text? The rest look good

Reply:
@ojwanganto 1-day old text added in the latest push. Thanks.

Reply:
Looks good. Thanks